Bs4 fix layout

stumbo opened the pull request:
Fix sidebar with author names to use all available space:
<img width="971" height="304" alt="image" src="https://github.com/user-attachments/assets/e2ea3571-fcfb-4b9d-b264-e625c5bfd538" />

Sidebar is responsive and will grow and shrink with window size.

stumbo committed:
Update grid class fot td-sidebar-toc to use all the available space.

The screen is divided into 12 columns, the existing layout was
only using 10.  Updated toc (where the taxonomy lives) to use
the remaining 2 columns.

masinter reviewed (commented):
I don't understand the git workflow here. There are 30 commits listing lots of changes, but the PR #300 shows only one file changed, layouts/bibliography/baseof.html with one trivial change.

Isn't it that approving and merging this PR will have no effect?

stumbo commented:
I cleaned up the branch - created a branch off main, cherry-picked the one commit, deleted the old bs4 branch, renamed my temp branch to bs4.., force pushed it to clean everything up.

pamoroso reviewed (approved):
The "All Authors" issue is now fixed.
